pkg/schedule/windowed_schedule_test.go

```diff
@@ -29,7 +29,7 @@ func TestWindowedSchedule(t *testing.T) {
 			err := w.Validate()
 			So(err, ShouldBeNil)
 
-			r := make([]Response, 0)
+			var r []Response
```

Comment:
Haha, yea I actually fixed one of these things a long time ago, just never touched the rest.